src/util/config/config.cpp

```diff
@@ -324,11 +324,22 @@ namespace dxvk {
   }
 
 
-  Config Config::getAppConfig(const std::string& appName) {
+  Config Config::getAppConfig(const std::string& appName, const std::string& parentDir) {
+    // Find configs by executable name
     auto appConfig = g_appDefaults.find(appName);
+
+    // Didn't find anything?
+    // Find configs by parent directory
+    if (appConfig == g_appDefaults.end()) {
+      appConfig = std::find_if(g_appDefaults.begin(), g_appDefaults.end(),
+        [&](const auto& appDir) -> bool {
+          return parentDir.ends_with(appDir.first);
```

Comment:
It looks like `::ends_with()` was added in C++20 [[1]](https://en.cppreference.com/w/cpp/string/basic_string/ends_with). In `meson.build` we set MSVC to use the latest standard available, but set `c++1z` for other compilers. It might be better to implement `::ends_with()` within `util_string` depending on what minimum compilers @doitsujin wants DXVK to support building with.

Reply:
Ah I was unaware of that.

On Discord we discussed moving to a regex approach on the full path